### PR TITLE
WIP: increase scope of test suite for integration tests

### PR DESCRIPTION
**Pull request by NightJar**
To pick up failures earlier, and make testing more manual. Also use
logical groupings so it's easier to categorise issues.
Also adjust config for Travis-CI to take advantage of this.

### CODE REVIEW

**Comment by NightJar** [comment]
https://travis-ci.org/creative-commoners/cwp-installer/builds/359296593

**Comment by NightJar** [comment]
Oh no! phpunit 5.7 doesn't support the multiple testsuite option! :'(

File: .travis.yml [hunk 1]
```diff
@@ -24,12 +24,14 @@ before_script:
 
   # Install composer dependencies
   - composer validate
+    # for silverstripe/config tests
+  - case "$PHPUNIT_TEST" in core*) composer require --no-update mikey179/vfsStream ^1.6; esac
```
Comment (robbieaverill) [inline .travis.yml:28]: Just require it outright, this syntax is harder to understand than Travis configuration generally is and I don't see much benefit

**Review by robbieaverill — CHANGES_REQUESTED** [review]
>Oh no! phpunit 5.7 doesn't support the multiple testsuite option! :'(

Perhaps just create new matrix entries for each test suite

File: .travis.yml (hunk 1 above)
Comment (NightJar) [reply .travis.yml:28]: Cool, I had it like that originally, but with trying to segment things I thought I should try and save those precious seconds :P
I'll revert :)

**Comment by robbieaverill** [comment]
Feel free to reopen this PR if you'd like to revisit it. The CWP kitchen sink recipe has some good examples of grouping tests into suites and running them in parallel in Travis. Cheers =)

**Pull request closed without merging**